ccl/cliccl: TestLoadShowIncremental failed

[(ccl/cliccl).TestLoadShowIncremental failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2802695&tab=buildLog) on [master@0796cbcd0493220e88982beebd45b9a077b713e9](https://github.com/cockroachdb/cockroach/commits/0796cbcd0493220e88982beebd45b9a077b713e9):

```
=== RUN   TestLoadShowIncremental
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental791271104
    test_log_scope.go:74: use -show-logs to present logs inline
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental904288754
    test_log_scope.go:74: use -show-logs to present logs inline
[load show incremental nodelocal://0/fooFolder --external-io-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/TestLoadShowIncremental682601129]
    load_test.go:196: 
        	Error Trace:	load_test.go:196
        	            				load_test.go:190
        	Error:      	Not equal: 
        	            	expected: "/fooFolder                     -                           2021-03-23T03:19:18Z\n/fooFolder/20210323/031918.99  2021-03-23T03:19:18Z        2021-03-23T03:19:18Z\n/fooFolder/20210323/031919.00  2021-03-23T03:19:19Z        2021-03-23T03:19:19Z\n"
        	            	actual  : "/fooFolder                     -                           2021-03-23T03:19:18Z\n/fooFolder/20210323/031918.99  2021-03-23T03:19:18Z        2021-03-23T03:19:18Z\n/fooFolder/20210323/031919.00  2021-03-23T03:19:18Z        2021-03-23T03:19:19Z\n"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -2,3 +2,3 @@
        	            	 /fooFolder/20210323/031918.99  2021-03-23T03:19:18Z        2021-03-23T03:19:18Z
        	            	-/fooFolder/20210323/031919.00  2021-03-23T03:19:19Z        2021-03-23T03:19:19Z
        	            	+/fooFolder/20210323/031919.00  2021-03-23T03:19:18Z        2021-03-23T03:19:19Z
        	            	 
        	Test:       	TestLoadShowIncremental
    testutils.go:210: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental904288754
    panic.go:636: -- test log scope end --
--- FAIL: TestLoadShowIncremental (1.30s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLoadShowIncremental PKG=./pkg/ccl/cliccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLoadShowIncremental.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/cliccl).TestLoadShowIncremental failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2803057&tab=buildLog) on [master@53bf501e233c337b9863755914d9c00010517329](https://github.com/cockroachdb/cockroach/commits/53bf501e233c337b9863755914d9c00010517329):

```
=== RUN   TestLoadShowIncremental
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental760997128
    test_log_scope.go:74: use -show-logs to present logs inline
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental388609402
    test_log_scope.go:74: use -show-logs to present logs inline
[load show incremental nodelocal://0/fooFolder --external-io-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/TestLoadShowIncremental608220817]
    load_test.go:196: 
        	Error Trace:	load_test.go:196
        	            				load_test.go:190
        	Error:      	Not equal: 
        	            	expected: "/fooFolder                     -                           2021-03-23T05:13:56Z\n/fooFolder/20210323/051356.99  2021-03-23T05:13:56Z        2021-03-23T05:13:56Z\n/fooFolder/20210323/051357.00  2021-03-23T05:13:57Z        2021-03-23T05:13:57Z\n"
        	            	actual  : "/fooFolder                     -                           2021-03-23T05:13:56Z\n/fooFolder/20210323/051356.99  2021-03-23T05:13:56Z        2021-03-23T05:13:56Z\n/fooFolder/20210323/051357.00  2021-03-23T05:13:56Z        2021-03-23T05:13:57Z\n"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -2,3 +2,3 @@
        	            	 /fooFolder/20210323/051356.99  2021-03-23T05:13:56Z        2021-03-23T05:13:56Z
        	            	-/fooFolder/20210323/051357.00  2021-03-23T05:13:57Z        2021-03-23T05:13:57Z
        	            	+/fooFolder/20210323/051357.00  2021-03-23T05:13:56Z        2021-03-23T05:13:57Z
        	            	 
        	Test:       	TestLoadShowIncremental
    testutils.go:210: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental388609402
    panic.go:617: -- test log scope end --
--- FAIL: TestLoadShowIncremental (0.24s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestLoadShowIncremental PKG=./pkg/ccl/cliccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLoadShowIncremental.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/cliccl).TestLoadShowIncremental failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2803058&tab=buildLog) on [master@53bf501e233c337b9863755914d9c00010517329](https://github.com/cockroachdb/cockroach/commits/53bf501e233c337b9863755914d9c00010517329):

```
=== RUN   TestLoadShowIncremental
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental923808668
    test_log_scope.go:74: use -show-logs to present logs inline
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental249425198
    test_log_scope.go:74: use -show-logs to present logs inline
[load show incremental nodelocal://0/fooFolder --external-io-dir=/go/src/github.com/cockroachdb/cockroach/artifacts/TestLoadShowIncremental686471861]
    load_test.go:196: 
        	Error Trace:	load_test.go:196
        	            				load_test.go:190
        	Error:      	Not equal: 
        	            	expected: "/fooFolder                     -                           2021-03-23T05:17:31Z\n/fooFolder/20210323/051731.99  2021-03-23T05:17:31Z        2021-03-23T05:17:31Z\n/fooFolder/20210323/051732.00  2021-03-23T05:17:32Z        2021-03-23T05:17:32Z\n"
        	            	actual  : "/fooFolder                     -                           2021-03-23T05:17:31Z\n/fooFolder/20210323/051731.99  2021-03-23T05:17:31Z        2021-03-23T05:17:31Z\n/fooFolder/20210323/051732.00  2021-03-23T05:17:31Z        2021-03-23T05:17:32Z\n"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -2,3 +2,3 @@
        	            	 /fooFolder/20210323/051731.99  2021-03-23T05:17:31Z        2021-03-23T05:17:31Z
        	            	-/fooFolder/20210323/051732.00  2021-03-23T05:17:32Z        2021-03-23T05:17:32Z
        	            	+/fooFolder/20210323/051732.00  2021-03-23T05:17:31Z        2021-03-23T05:17:32Z
        	            	 
        	Test:       	TestLoadShowIncremental
    testutils.go:210: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLoadShowIncremental249425198
    panic.go:636: -- test log scope end --
--- FAIL: TestLoadShowIncremental (1.04s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-race -parallel=4

```
make stressrace TESTS=TestLoadShowIncremental PKG=./pkg/ccl/cliccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLoadShowIncremental.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/ccl/cliccl/load_test.go
+++ b/pkg/ccl/cliccl/load_test.go
@@ -183,7 +183,7 @@ func TestLoadShowIncremental(t *testing.T) {
 	w := tabwriter.NewWriter(&buf, 28 /*minwidth*/, 1 /*tabwidth*/, 2 /*padding*/, ' ' /*padchar*/, 0 /*flags*/)
 	fmt.Fprintf(w, "/fooFolder	-	%s\n", ts[0].GoTime().Format(time.RFC3339))
 	fmt.Fprintf(w, "/fooFolder%s	%s	%s\n", expectedIncFolder, ts[0].GoTime().Format(time.RFC3339), ts[1].GoTime().Format(time.RFC3339))
-	fmt.Fprintf(w, "/fooFolder%s	%s	%s\n", expectedIncFolder2, ts[2].GoTime().Format(time.RFC3339), ts[2].GoTime().Format(time.RFC3339))
+	fmt.Fprintf(w, "/fooFolder%s	%s	%s\n", expectedIncFolder2, ts[1].GoTime().Format(time.RFC3339), ts[2].GoTime().Format(time.RFC3339))
 	if err := w.Flush(); err != nil {
 		t.Fatalf("TestLoadShowIncremental: flush: %v", err)
 	}

